packages/ember-glimmer/lib/components/link-to.js

```diff
@@ -594,17 +594,19 @@ const LinkComponent = EmberComponent.extend({
   }),
 
   transitioningIn: computed('active', 'willBeActive', function computeLinkToComponentTransitioningIn() {
-    let willBeActive = get(this, 'willBeActive');
-    if (typeof willBeActive === 'undefined') { return false; }
-
-    return !get(this, 'active') && willBeActive && 'ember-transitioning-in';
+    if (get(this, 'willBeActive') === true && !get(this, 'active')) {
```

Comment:
I'm wondering why the mix of comparing to a boolean on the first clause, but using `!` on the second one?